dpdk/vr_dpdk_knidev.c

```diff
@@ -324,6 +324,8 @@ vr_dpdk_knidev_init(struct vr_interface *vif)
         else
             /* ...so we use os_idx instead */
             port_id = vif->vif_os_idx;
+    } else if (vif->vif_type == VIF_TYPE_MONITORING) {
+            port_id = 0; /* we always use DPDK port 0 */
```

Comment:
I think it is fine if port 0 is always allocated by DPDK if an interface is present. Please add a TODO comment with the above info though.